fields/types/list/ListField.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/jsx-no-bind */
+
+import assign from 'object-assign';
+import React from 'react';
+import Field from '../Field';
+import Domify from 'react-domify';
+
+import { Fields } from 'FieldTypes';
+import { Button, GlyphButton } from '../../../admin/client/App/elemental';
+import InvalidFieldType from '../../../admin/client/App/shared/InvalidFieldType';
+
+function generateId () {
+	return Math.random().toString(36).substr(2, 9);
```

Comment:
I don't understand why this is a random. If you "win the lottery", you get a conflicting id.
Can't it be an monotonously increasing integer, perhaps based on a uuid given by the server if it can't start at 0?